### DISCUSSION TITLE
not opening nvimtree when using neotree at the same time

### DISCUSSION THREAD

**Discussion by cxwx** [nvim-tree/nvim-tree.lua#3012] [Q&A] [answered]
Hi all:
  I'm using neotree and nvimtree at the same time, when I tried
  ```
  nvim .
  ```
  
![截屏2024-11-24 19 02 38](https://github.com/user-attachments/assets/d6d0265e-80fd-4970-9489-66540b3d9982)

It will use nvimtree and neotree,

I'd love to not opening nvimtree when starting nvim with a directory.
How to do?

**Comment by alex-courtis** [comment, marked as answer]
You could disable `:help nvim-tree-opts-hijack-directories`

What's the use case for two trees? I predict that they will disrupt each other's functionality.

**Reply by cxwx** [reply]
thanks

**Comment by cxwx** [comment]
thanks
